libraries/stdlib/js/src/kotlin/collections/ArrayList.kt

```diff
@@ -71,7 +71,7 @@ public actual open class ArrayList<E> internal constructor(private var array: Ar
         checkIsMutable()
         if (elements.isEmpty()) return false
 
-        array += elements.toTypedArray<Any?>()
+        dummy.push.apply(array, elements.toTypedArray<Any?>())
```

Comment:
AFAIK, there was some limitation on how many parameters can be spread using this approach, so it may fail on large `elements` collection